services/src/main/java/cz/muni/fi/pa165/facade/UserFacadeImpl.java

```diff
@@ -71,9 +71,12 @@ public void updateUser(UserDTO user) {
     }
 
     @Override
-    public Enum<AuthenticateUserStatus> authenticate(UserAuthenticateDTO userAuthenticateDTO) {
+    public UserDTO authenticate(UserAuthenticateDTO userAuthenticateDTO) {
```

Comment:
Tohle vubec nedava smysl. My chceme, aby to vracelo enum a ne UserDTO!

Ty namapujes UserAuthenticateDTO na Users, pouzijes to v service a pokud je to OK, tak to premapujes na UserDTO, ve kterem je nastaveny porad jenom email a heslo. Kdyz to neni validni, tak ztratis rozdil, jestli je spatne heslo nebo email.

Vracej porad jenom vystup service a veskerou logiku, kdyz nastane nejaky stav enumu, delej jinde